.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,10 +9,16 @@ assignees: ''
 
 <!-- NOTE:
 
+Those information will be helpful for us to investigate the problem. So please, follow the steps we provided exactly when submitting your issue, and do not remove the issue template.
+
 If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
 
 If you want help on your bug, please also send us the git repository (GitHub, GitLab, Bitbucket etc...) where your hexo code is stored. It would greatly help. If you prefer not to have your hexo code out in public, please upload to a private GitHub repository and grant read-only access to hexojs/core.
 
+Please DO NOT CONTAIN SECRET INFORMATION (likes password or GitHub Personal Access Token) when submitting a new issue.
```

Comment:
"Please take extra precaution not to attach any secret environment variables (e.g. password and token)."